apps/admin/integration-testing/playwright.config.ts

```diff
@@ -12,6 +12,7 @@ dotenv.config({ path: resolve(__dirname, '.env') });
  */
 export default defineConfig({
   testDir: './e2e',
+  timeout: 60_000,
```

Comment:
Might need to go higher - seeing flaky timeouts in the latest CircleCI run

Reply:
Where?

Reply:
If you open up the `admin-integration-test` -> `Test` step, it has the logs of two timed out attempts and a third successful one: https://app.circleci.com/pipelines/github/votingworks/vxsuite/17636/workflows/807e1e8b-f7be-49e6-a307-bb8889e440a9/jobs/706679

Reply:
good catch! i think the issue is not the timeout but a point where the test can sometimes hang, which I can fix.

Reply:
Side note: also now wondering if retries got turned on automatically in newer versions of Playwright (or if I've just never noticed that before as the default setting) - it's great for getting through CI without intervention, but not so great for our visibility into flaky tests, so I'm torn on whether or not we should leave it on